.specify/scripts/bash/update-agent-context.sh

```
@@ -0,0 +1,781 @@
+
+# Enable strict error handling
+set -u
+set -o pipefail
+
+#==============================================================================
+# Configuration and Global Variables
+#==============================================================================
+
+# Get script directory and load common functions
+SCRIPT_DIR="$(CDPATH="" cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+source "$SCRIPT_DIR/common.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./common.sh was not specified as input (see shellcheck -x).

(SC1091)